src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -6913,13 +6913,20 @@ subroutine atm_bdy_set_scalars_work( scalars_driving, scalars_new, &
 
       !---
 
+      MPAS_ACC_TIMER_START('atm_init_coupled_diagnostics [ACC_data_xfer]')
```

Comment:
Could you update the string in timer calls to `atm_bdy_set_scalars_work` in all four instances.

Reply:
This has been addressed by force-push [982098f](https://github.com/MPAS-Dev/MPAS-Model/commit/982098ffad7242b7f83030ebcbe27efec19ee5c5) to [7d1d937](https://github.com/MPAS-Dev/MPAS-Model/commit/7d1d937a5b311db76885086c4dd2d1616fc1fe0f)